sym_api_client_python/clients/sym_bot_client.py

```diff
@@ -166,10 +166,10 @@ def execute_rest_call(self, method, path, **kwargs):
             logging.debug(type(err))
             logging.debug('ensure pod/agent subdomains are correct')
             raise
-
+        #TODO maybe a 400 response call need to stop datafeed and create a new one
```

Comment:
I am not in favour of pushing TODO comments. Please remove it if you are not going to implement what you suggested.